root/static/scripts/common/i18n/localizeInstrumentName.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {l_instruments, lp_instruments} from './instruments';
+
+function localizeInstrumentName(instrument: InstrumentT) {
+  if (instrument.comment) {
+    return lp_instruments(instrument.name, instrument.comment);
```

Comment:
I cannot understand what it does or even what it is intended to do.

Reply:
It is used for translators to be able to distinguish between different instruments of the same name (in English) that might have different names in their language.

Reply:
So you actually fixed a bug.